ibis/backends/csv/__init__.py

```diff
@@ -56,12 +64,8 @@ def table(self, name, path=None, schema=None, **kwargs):
 
         return table
 
-    @property
-    def version(self):
-        return parse_version(pd.__version__)
 
-
-@pre_execute.register(ops.Selection, CSVClient)
+@pre_execute.register(ops.Selection, (Backend, CSVClient))
```

Comment:
I guess this will go away after everything becomes a backend?

Reply:
Yes, when client classes are deleted and fully replaced by backend classes (with some luck in the next PR after this one), all those will register only backend classes, not both.